src/sayt/autocomplete.ts

```diff
@@ -0,0 +1,25 @@
+import { QueryTimeAutocompleteConfig } from 'sayt';
+import { AutocompleteResultGroup } from './results';
+
+/** The name of the event fired when an autocomplete request is to be made. */
```

Comment:
To be consistent with the `SEARCH_REQUEST` event, could change this to something like: 'The name of the event fired to request autocomplete data'

Reply:
Updated.